contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -1264,6 +1342,50 @@ export const AssessmentItem = new Resource({
       ...id,
     });
   },
+
+  delete(id) {
+    return this.transaction('rw', TABLE_NAMES.CONTENTNODE, () => {
+      const nodeId = id[0];
+      return this.table.delete(id).then(data => {
+        // Update assessment item count
+        return this.transaction('rw', TABLE_NAMES.CONTENTNODE, () => {
+          Dexie.currentTransaction.source = IGNORED_SOURCE;
+          return ContentNode.get(nodeId).then(node => {
+            if (node) {
+              return ContentNode.update(node.id, {
+                assessment_item_count: (node.assessment_item_count || 1) - 1,
```

Comment:
I'm not sure why, but for some reason this change is getting propagated to the backend, not ignored.

Going to merge, I don't see anything wrong with the code here - will file a follow up issue instead.